src/main/java/io/jenkins/plugins/gitlabbranchsource/retry/TagsApiRetryWrapper.java

```diff
@@ -0,0 +1,30 @@
+package io.jenkins.plugins.gitlabbranchsource.retry;
+
+import java.util.List;
+import org.gitlab4j.api.GitLabApiException;
+import org.gitlab4j.api.TagsApi;
+import org.gitlab4j.api.models.Tag;
+
+import static io.jenkins.plugins.gitlabbranchsource.retry.RetryHelper.withRetry;
+
+public class TagsApiRetryWrapper {
+    private final TagsApi _tagsApi;
```

Comment:
This is not C# code, please follow Java code standards.
```suggestion
    private final TagsApi tagsApi;
```